feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -149,122 +162,156 @@ fun PreviewScreen(
-                            Spacer(
-                                modifier = Modifier
-                                    .fillMaxHeight()
-                                    .weight(1f)
-                            )
-                        }
-
-                        VideoRecordingState.INACTIVE -> {
-                            FlipCameraButton(
-                                modifier = Modifier
-                                    .weight(1f)
-                                    .fillMaxHeight(),
-                                onClick = { viewModel.flipCamera() },
-                                // enable only when phone has front and rear camera
-                                enabledCondition =
-                                previewUiState.currentCameraSettings.isBackCameraAvailable &&
-                                    previewUiState.currentCameraSettings.isFrontCameraAvailable
-                            )
-                        }
-                    }
-                    val multipleEventsCutter = remember { MultipleEventsCutter() }
-                    /*todo: close quick settings on start record/image capture*/
-                    CaptureButton(
-                        modifier = Modifier
-                            .testTag(CAPTURE_BUTTON),
-                        onClick = {
-                            multipleEventsCutter.processEvent { viewModel.captureImage() }
-                        },
-                        onLongPress = { viewModel.startVideoRecording() },
-                        onRelease = { viewModel.stopVideoRecording() },
-                        videoRecordingState = previewUiState.videoRecordingState
-                    )
-                    /* spacer is a placeholder to maintain the proportionate location of this
-                     row of UI elements. if you want to  add another element, replace it with ONE
-                     element. If you want to add multiple components, use a container
-                     (Box, Row, Column, etc.)
-                     */
-                    Spacer(
-                        modifier = Modifier
-                            .fillMaxHeight()
-                            .weight(1f)
-                    )
+                    /*TODO("Place other components here") */
                 }
             }
-            // displays toast when there is a message to show
-            if (previewUiState.toastMessageToShow != null) {
-                ShowTestableToast(
-                    modifier = Modifier
-                        .testTag(previewUiState.toastMessageToShow!!.testTag),
-                    toastMessage = previewUiState.toastMessageToShow!!,
-                    onToastShown = viewModel::onToastShown
-                )
-            }
+        }
+        // displays toast when there is a message to show
+        if (previewUiState.toastMessageToShow != null) {
+            ShowTestableToast(
```

Comment:
Is the previewUiState.toastMessageToShow mutable? 

Should add a `requireNotNull` for previewUiState.toastMessageToShow

Reply:
yes, it stays null when there isn't anything to show. when it is populated, the toast is shown and it returns to null. the "testable" part of the toast is an invisible box that is present for as long as the `toastMessageToShow` isn't null.

Thinking about it again, maybe toastMessageToShow should be a list or queue should other units want to call a toast in quick succession... but i don't think toasts should be used frequently anyways since they're a bit invasive on the UI